source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,229 @@
+``find()`` method to  run a text search. 
+
+When you execute the ``find()`` method, MongoDB runs a text search on
+all the fields indexed with the text index on the collection. MongoDB
+returns documents that contain one or more of the search terms and a
+:ref:`relevance score for each result <sorts-crud-text-search>`. 
+
+Specify Options
+~~~~~~~~~~~~~~~
+
+You can include ``TextSearchOptions`` as the second parameter of the
+``Filters.text()`` method to specify text search options such as case
+sensitivity. By default, text searches run without case sensitivity
+which means the search matches lowercase and uppercase values.
+
+To specify a case sensitive search, use the following snippet:
+
+.. code-block:: java
+
+   TextSearchOptions options = new TextSearchOptions().caseSensitive(true);
+   Bson filter = Filters.text("SomeText", options);
+
+For more information on the method and class in this section, see the following API documentation:
+
+- :java-core-api:`Filters.text() <com/mongodb/client/model/Filters.html#text(java.lang.String)>`
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass a term as a string to the ``Filters.text()`` method to specify the
+term in your text search. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin termExample
+   :end-before: end termExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+To match multiple terms in your text search, separate each term
+with spaces in the ``Filters.text()`` builder method. When you pass the 
+text search query returned by the builder method to the
+``find()`` method, it returns documents that match any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+terms "fate" or "7".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin multipleTermExample
+   :end-before: end multipleTermExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a phrase with escaped quotes to the ``Filters.text()`` method to
```

Comment:
From a Jasmin persona reader, I wonder whether "escaped quotes" might be jargon which should be defined/demonstrated as you might do for a term. E.g.
Escaped quotes are double quote characters preceded by a backslash character.

And then specify that the positioning of the escaped quotes.
"If you don't add escaped quotes around the phrase, the method performs..."